network-store-client/src/main/java/com/powsybl/network/store/client/RestNetworkStoreClient.java

```diff
@@ -152,6 +152,12 @@ public void cloneNetwork(UUID networkUuid, int sourceVariantNum, int targetVaria
                 networkUuid, sourceVariantNum, targetVariantNum, targetVariantId);
     }
 
+    @Override
+    public void cloneNetwork(UUID networkUuid, List<Resource<NetworkAttributes>> networkResources) {
```

Comment:
I would have expected UUID cloneNetwork(UUID networkUuid)
Why are you passing a list of resources to the clone?